src/core/map.js

```diff
@@ -1194,11 +1226,14 @@ geo.map = function (arg) {
    * Cancel any existing transition.  The transition will send a cancel event
    * at the next animation frame, but no further activity occurs.
    *
+   * @param {string} source option cause of the cancel.
```

Comment:
Sounds good.  The [jsdoc syntax](http://usejsdoc.org/tags-param.html) for optional arguments is typically `@param {string} [source]`.